### DISCUSSION TITLE
Set links at the top resources bar to go to the preferred market view

### DISCUSSION THREAD

**Discussion by trs4ece** [BlackAsLight/DocScripts#5] [Ideas]
The market view links are pretty nice, but when looking at the Both view, we have to scroll all the way back up to the top to view trades for another resource. If you could update the resource links in the nation resources bar at the top of the screen to also jump to the preferred Market View, this would be a time saver.

**Comment by BlackAsLight** [comment]
I think it would be better to create a new script that runs on every page to update these types of links and some things on the sidebar. So you can get the same results, regardless of the page.

**Reply by trs4ece** [reply]
Sure. I've got a separate script which helps with warchest tracking. I just updated the links through that - https://greasyfork.org/en/scripts/401046-resource-bar-helper
